pkg/volume/glusterfs/glusterfs.go

```diff
@@ -644,12 +644,12 @@ func (d *glusterfsVolumeDeleter) Delete() error {
 	}
 	err = cli.VolumeDelete(volumeID)
 	if err != nil {
-		glog.Errorf("error when deleting the volume :%v", err)
+		glog.Errorf("error when deleting the volume(%s): %v", volumeName, err)
```

Comment:
@liubin Can you please use `[]` instead of `()` in all the error messages?

Reply:
ok